docs/Doxyfile.in

```diff
@@ -54,5 +54,6 @@ IMAGE_PATH             = @PROJECT_SOURCE_DIR@/docs/graphics
 TEMPLATE_RELATIONS     = YES
 HAVE_DOT               = YES
 GENERATE_XML           = YES
+USE_MATHJAX            = YES # default NO
```

Comment:
why is this necessary? Maybe add a comment in the doxyfile

Reply:
Because of the math I added to the header of `AxilrodTellerFunctor.h` ;)